PAS-55 Add components into static login page

Samantha-norrie opened the pull request:
Hi all :) All of the components needed for the login page are now implemented. Since my last draft PR for this ticket, I've added in the text input component, changed the link formatting, and added a shadow under the login button. I am currently unsure about 2 things:
1. How I should be adding in placeholders within the login page for the text input component? Edit: After running the tests I've realized that I didn't handle onSubmit or emailError for the login button, which I guess fits into this question, too!
2. Does the shadow cause the login button to be ever so slightly uncentered? I've been staring at my screen for so long and now can't seem to see if I'm imagining things or if it is really off by a bit!

Please let me know if there is anything else that you'd like me to change!

daelynj commented on webpack.config.js:
```diff
@@ -28,7 +28,7 @@ module.exports = {
     devServer: {
         contentBase: path.join(__dirname, 'dist'),
-        compress: true,
+        compress: false,
```
This seems unrelated to linting, is that true?

BraidenCutforth replied:
Fixed

daelynj commented on .eslintrc.js:
```diff
@@ -16,6 +16,8 @@ module.exports =  {
     rules:  {
         '@typescript-eslint/explicit-function-return-type': 'off',
     },
+    reportUnusedDisableDirectives: true,
```
Can you elaborate on why you would want to set these two statements to true? Might be helpful for people (..me) unfamiliar with configuring ESLint if this was explained in a commit message.

BraidenCutforth replied:
I added a comment, but to explain more thoroughly here, the reportUnusedDisableDirectives will make sure that all of our inline disable directives such as "eslint ignore-next-line" will actually be causing eslint to ignore something. We have set rules in eslint, such as camelCase being enforced and sometimes it makes sense to ignore the styling rules. In order to ignore the rule in a single place, a comment is used. If the code changes though and follows the rules, we want to make sure that we are also cleaning up those comments.

The root option just tells eslint that this is the root configuration file and that it shouldn't keep traversing up the directory tree to find other config files. Leads to more consistent linting across machines

daelynj replied:
Nice! Thanks for explaining.

Commit:
PAS-55 beginning to add components to login page

Commit:
PAS-55 beginning to add components to login page

Commit:
Adding change to not lose while switching to other ticket

Commit:
Adding change to not lose while switching to other ticket

Commit:
Merge branch 'PAS-55' of https://github.com/VikeLabs/Passr into PAS-55

Commit:
PAS-55 added in input components and added shadow around login button

Commit:
PAS-55 removed logo import

BraidenCutforth committed:
PAS-55: Functionality fix up

- Add placeholder text to text input
- Add placeholder for username and password
- Remove nested container and shadow on login button
- Attach email err to email input
- Set log in button onClick to onSubmit

BraidenCutforth commented:
Looks great!
